tests/filesystem.go

```diff
@@ -301,10 +301,10 @@ func formatPartition(ctx context.Context, partition *types.Partition) error {
 		return writePartitionData(partition.Device, partition.FilesystemImage)
 	default:
 		if partition.FilesystemType == "blank" ||
-			partition.FilesystemType == "" {
+			partition.FilesystemType == "" || partition.FilesystemType == "none" {
```

Comment:
nit: if gofmt will let you, wrap the second condition onto the next line for consistency with the `"blank"` check.

Reply:
This is now in the correct commit, but is back to the weird line-wrapping.

Reply:
Fixed